src/polar/Radar.tsx

```diff
@@ -227,7 +227,7 @@ export class Radar extends PureComponent<Props, State> {
   renderDots(points: RadarPoint[]) {
     const { dot, dataKey } = this.props;
     const baseProps = filterProps(this.props);
-    const customDotProps = filterProps(dot);
+    const customDotProps = filterProps(dot, true);
```

Comment:
Should we make the second parameter non optional to require setting it everywhere? 
Not required for this fix, but wondering whether there are other cases where the events should exist but do not. 🤔

Reply:
I always prefer to pass options as objects so they are named. Ordered params can get confusing, but yeah making it required wouldn't be the worst thing ever I think. @AnujSharma141 would you like to do that as follow up?

Reply:
I also think it might be possible to get the same bug again if someone misses to add the parameter. We might need to see the occasions where the filterProps function is used but I think this can be done.

Reply:
Make it required, then you'll get errors anywhere it isn't set. Resolve the errors correctly and we should be good